src/http/modules/ngx_http_upstream_session_sticky_module.c

```diff
@@ -533,12 +533,6 @@ ngx_http_session_sticky_get_cookie(ngx_http_request_t *r)
         & (NGX_HTTP_SESSION_STICKY_PREFIX | NGX_HTTP_SESSION_STICKY_INDIRECT))
     {
         cookie->len -= (end - st);
-        if (cookie->len == 0) {
```

Comment:
hi @detailyang,
 @dinic, author of this module, has fixed this bug in our internal tengine, his patch is as following:

```
        if (cookie->len == 0) {
            cookies[i]->hash = 0;
            return NGX_OK;
        }
```